src/createServer.js

```diff
@@ -1,11 +1,33 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const userController = require('./controllers/user.controller.js');
+const expensesController = require('./controllers/expense.controller.js');
+
+const userService = require('./services/user.service.js');
+const expenseService = require('./services/expense.service.js');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  userService.resetUsers();
+  expenseService.resetExpenses();
+
+  app.use(express.json());
+
+  app.get('/users', userController.getAll);
+  app.get('/users/:id', userController.getById);
+  app.post('/users', userController.create);
+  app.delete('/users/:id', userController.remove);
+  app.patch('/users/:id', userController.patch);
+
+  app.get('/expenses', expensesController.getAll);
+  app.get('/expenses/:id', expensesController.getById);
+  app.post('/expenses', expensesController.create);
+  app.delete('/expenses/:id', expensesController.remove);
+  app.patch('/expenses/:id', expensesController.patch);
```

Comment:
Don't keep all your routes in the same place, create separate router for users and expenses, and move them into separate files

Reply:
I implement router 🤝.